build/Common.prod.props

```diff
@@ -23,7 +23,6 @@
     <PropertyGroup>
       <RevisionNumber>0</RevisionNumber>
       <RevisionNumber Condition="$(MinVerVersion.Split(`.`).Length) == 4">$(MinVerVersion.Split(`.`)[3])</RevisionNumber>
-      <AssemblyVersion>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch).$(RevisionNumber)</AssemblyVersion>
       <Version>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch).$(RevisionNumber)</Version>
       <FileVersion>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch).$(RevisionNumber)</FileVersion>
```

Comment:
This target is still required, as the default Minver does not include revision number in the FileVersion. The dotnet recommendation is to *include* revision number - https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/versioning#assembly-file-version